### ISSUE TITLE
Remove Block Editor Files

### ISSUE THREAD

**Issue by bahiirwa** [ClassicPress-research/classic-forms-8#4]
ClassicPress does not support the Block Editor so this will be redundant code if left in the codebase.

**Comment by timbocode** [comment]
Yes, all block editor code must be removed before the first release.

**Comment by ginsterbusch** [comment]
For the initial fork, ie. to have something that just works, the commented out part should be enough, but for the actual stable 1.0 release it definitely needs to go.

**Comment by bahiirwa** [comment]
Agreed.
